query-builder/src/main/java/com/datastax/oss/driver/api/querybuilder/update/OnGoingAssignment.java

```diff
@@ -0,0 +1,488 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.querybuilder.update;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.querybuilder.QueryBuilderDsl;
+import com.datastax.oss.driver.api.querybuilder.term.Term;
+import java.util.Arrays;
+
+public interface OnGoingAssignment {
```

Comment:
nit:  This is named `OnGoingAssignment`, while the rest of the `OngoingXXX` classes use `Ongoing`

Reply:
👍 good catch, "ongoing" is a single word so `Ongoing` is the correct capitalization, fixing it.